### PR TITLE
Removed checkboxes, changed image layout. Refactoring #625

### LINKED ISSUE #625: (SP: 3) Refactor news form
Checkboxes should be removed, view language panel and we have to see image upload due to our new UI firstly
Screenshot 2021-03-10 at 14 32 25

We have to see image upload like this: 
Screenshot 2021-03-10 at 14 40 54
Screenshot 2021-03-10 at 14 40 48

### PR DESCRIPTION
**Pull request by grebiniuk**
- Removed checkboxes, 
- Showed both EN and UA news info.
- Added and fixed Back and Save buttons to the top of the page.
- Added new image upload buttons with preview

#### Screenshots

Original             |  Updated
:-------------------------:|:-------------------------:
image | image

Resolves #625 
Closes #625 

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅ All tests passed locally
- [x] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

**Comment by Kiyooooko** [comment]
pls add little margin after buttons
Screenshot 2021-03-23 at 18 09 39

**Comment by Kiyooooko** [comment]
pls make git pull origin development ad fix this conflicts
Screenshot 2021-03-23 at 18 10 19

**Comment by Kiyooooko** [comment]
why you dublicate buttons?
Screenshot 2021-03-23 at 18 13 37

File: src/components/forms/news-form/news-form.js [hunk 1]
```diff
@@ -147,65 +135,71 @@ const NewsForm = ({ id, newsArticle, editMode }) => {
...
-            title={SAVE_TITLE}
-          />
-        </div>
-        <Box my={3}>
-          <div className={styles.imageUploadAvatar}>
-            <ImageUploadContainer
-              handler={handleImageLoad}
-              buttonLabel={AUTHOR_PHOTO}
-            />
+        {navButtons}
```
Comment (Kiyooooko) [inline src/components/forms/news-form/news-form.js:153]: pls remove one this component

File: src/components/forms/news-form/news-form.js [hunk 2]
```diff
@@ -147,65 +135,71 @@ const NewsForm = ({ id, newsArticle, editMode }) => {
...
+                  )}
+                </div>
+              </div>
+            </div>
+          </Paper>
+        </Grid>
+        {map(languages, (lang) => (
+          <LanguagePanel lang={lang} inputOptions={inputOptions} key={lang} />
+        ))}
+        {navButtons}
```
Comment (Kiyooooko) [inline src/components/forms/news-form/news-form.js:201]: or this

**Comment by Kiyooooko** [comment]
after your changes we can't save news
Screenshot 2021-03-23 at 18 16 47

**Comment by grebiniuk** [comment]
> pls add little margin after buttons
> Screenshot 2021-03-23 at 18 09 39
Margin bottom here is the same as these buttons margin top in the bottom.

**Comment by grebiniuk** [comment]
> pls make git pull origin development ad fix this conflicts
> Screenshot 2021-03-23 at 18 10 19

Ok

**Comment by grebiniuk** [comment]
> why you dublicate buttons?
> Screenshot 2021-03-23 at 18 13 37

Improved UI, there are a lot of inputs and you need to scroll a lot to go backwards or save the news.
As an option, I can make the position fixed as we have save/delete while editing a product.

File: src/components/forms/news-form/news-form.js (hunk 1 above)
Comment (grebiniuk) [reply src/components/forms/news-form/news-form.js:153]: Improved UI, there are a lot of inputs and you need to scroll a lot to go backwards or save the news.
As an option, I can make the position fixed as we have save/delete while editing a product.

File: src/components/forms/news-form/news-form.js (hunk 2 above)
Comment (grebiniuk) [reply src/components/forms/news-form/news-form.js:201]: Improved UI, there are a lot of inputs and you need to scroll a lot to go backwards or save the news.
As an option, I can make the position fixed as we have save/delete while editing a product.

**Comment by grebiniuk** [comment]
> after your changes we can't save news
> Screenshot 2021-03-23 at 18 16 47

Will have a look, thank you!

**Comment by MMashivskij** [comment]
Input validation work after submit click, only.
It should work after unfocus.

image

**Comment by MMashivskij** [comment]
The button "Зберегти" is still disable, even if input value is correct.

image
image

**Comment by grebiniuk** [comment]
> The button "Зберегти" is still disable, even if input value is correct.
> 
> image
> image

Reproduced on staging.

**Comment by grebiniuk** [comment]
> Input validation work after submit click, only.
> It should work after unfocus.
> 
> image

Reproduced on staging

**Comment by grebiniuk** [comment]
> pls add little margin after buttons
> Screenshot 2021-03-23 at 18 09 39

fixed

**Comment by grebiniuk** [comment]
> pls make git pull origin development ad fix this conflicts
> Screenshot 2021-03-23 at 18 10 19

fixed

**Comment by grebiniuk** [comment]
> why you dublicate buttons?
> Screenshot 2021-03-23 at 18 13 37

fixed

File: src/components/forms/news-form/news-form.js (hunk 2 above)
Comment (grebiniuk) [reply src/components/forms/news-form/news-form.js]: Made position fixed for save/delete buttons

**Review by MMashivskij — APPROVED** [review]

**Review by Haytham1337 — APPROVED** [review]

**Pull request merged**